lang/LangSource/PyrLexer.cpp

```
@@ -2273,7 +2273,10 @@ SCLANG_DLLEXPORT_C void runLibrary(PyrSymbol* selector) {
         } else {
             postfl("Library has not been compiled successfully.\n");
         }
-    } catch (std::exception& ex) {
+    } catch (const FatalInterpreterError& er) {
+        error("A fatal interpreter error has occured. Reason: %s\n", er.what());
+        std::rethrow_exception(std::current_exception());
```

If you want to rethrow the current exception, you can simply write `throw;` (without any exception object!)

---

Oh I didn't know that! I'll change it when I get the change. Thanks!